tensorflow_serving/g3doc/serving_advanced.md

```diff
@@ -143,7 +143,17 @@ With all these, `ServerCore` internally does the following:
   the `SessionBundleSourceAdapter`.
 
 Whenever a new version is available, this `AspiredVersionsManager` always
-unloads the old version and replaces it with the new one. If you want to
+unloads the old version and replaces it with the new one. The default behavior means that
+there will only be one version of the model served at a given time (and that version will be the 
+latest version). The latest version is determined from the model folder name which should be numeric.
+If you would like to serve all the versions available in `model_base_path`, you can set the following environment 
+variable before compiling Tensorflow Serving from source.
+
+~~~
+export TF_SERVING_MULTIPLE_VERSIONS=1
```

Comment:
I think for those that are starting out to play with TS serving, they most likely do not need this feature. But once they get to productionizing their model serving pipeline, i believe versioning and A/B testing is generally becomes fairly important. 

Is there other places where the features of TS serving is documented besides in the tutorials? Or usually it's expected for users to read the codebase ?